datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -839,4 +839,24 @@ CREATE TABLE dcsa_im_v3_0.operations_event (
 
 ALTER TABLE dcsa_im_v3_0.operations_event ADD PRIMARY KEY (event_id);
 
+-- DDT-1180 - message routing
```

Comment:
Please move this to 05 as it is not part of the official IM

Reply:
Done